README.md

```diff
@@ -226,7 +226,9 @@ Place holder `<interface-name>` needs to be replaced with the real name,
 of course.
 You can also specify multiple interface arguments here which need to be
 either all active for online state, or only one of them if argument
-`--any` is added, too.
+`--any` is added, too.<br>
+Set `OMNECT_WAIT_ONLINE_TIMEOUT_IN_SEC` to configure the timeout in seconds, default timeout is 5 minutes.
+

```

Comment:
should we mention "0" means disabled?
should we link https://github.com/omnect/omnect-device-service?tab=readme-ov-file#configure-wait-online-reboot-timeout?

Reply:
@JanZachmann should we?

Reply:
i decided to document the value of 0, but not include the link, since i see no additional information there. (it links back to this Readme).